ecmascript/codegen/src/lib.rs

```diff
@@ -1216,6 +1217,20 @@ impl<'a> Emitter<'a> {
         }
     }
 
+    #[emitter]
+    fn emit_static_block(&mut self, n: &StaticBlock) -> Result {
+        self.emit_leading_comments_of_span(n.span(), false)?;
+        keyword!("static");
+        punct!("{");
```

Comment:
Maybe `formatting_space!()` here?
It's not important, though.

Reply:
`f3e295f` :+1: